.lintrunner.private.toml

```diff
@@ -0,0 +1 @@
+.lintrunner.toml
```

Comment:
I'm not sure why this symlink is needed, but I have needed it to run `lintrunner` on my workstation also.

Reply:
hmm. this shouldn't be needed. It's probably some funny script issue for lintrunner.

Meanwhile, can we change the command to `lintrunner --config .lintrunner.toml XXXXXXX`